code/src/components/HoverOverlay.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import './hover.css'
+
+export const HoverOverlay = () => {
+  return (
+
+    <div className='overlay'>
```

Comment:
You are using `"` over `'` when it comes to the markup, so it would be nice if this follows the same rule
```suggestion 
    <div className="overlay">
```